testlib/src/main/java/io/spine/testing/Tests.java

```diff
@@ -179,9 +216,15 @@ public static void assertMatchesMask(Message message, FieldMask fieldMask) {
         // Assert that values match the field mask.
         for (FieldDescriptor field : fields) {
             if (field.isRepeated()) {
-                continue;
+                boolean maskHasSuchField = paths.contains(field.getName());
+                if (maskHasSuchField) {
+                    List repeatedFieldValue = (List) message.getField(field);
```

Comment:
Please use `List<?>` instead. Raw types exist in Java only for compatibility reasons and should not be used in new code.